shortcircuit-api/src/main/java/com/powsybl/shortcircuit/FaultResult.java

```diff
@@ -37,8 +50,25 @@ public String getId() {
     /**
      * Value of the 3-phase short-circuit current for this fault (in A).
      */
-    public float getThreePhaseFaultCurrent() {
+    public double getThreePhaseFaultCurrent() {
         return threePhaseFaultCurrent;
     }
 
+    /**
+     * List of contributions to the three phase fault current of each connectable connected to the equipment
+     */
+    public List<FeederResult> getFeederResults() {
+        return feederResults;
+    }
```

Comment:
Good practice for getters of lists: never directly pass the lists to prevent them to be modified without security. Always pass a unmodifiable copy: ```Collections.unmodifiableList(feederResults)```